x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_event_log_list_kpi.tsx

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect, useState, useMemo, useRef } from 'react';
+import { i18n } from '@kbn/i18n';
+import datemath from '@kbn/datemath';
+import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiIconTip, EuiStat, EuiSpacer } from '@elastic/eui';
+import { IExecutionKPIResult } from '@kbn/alerting-plugin/common';
+import {
+  ComponentOpts as RuleApis,
+  withBulkRuleOperations,
+} from '../../common/components/with_bulk_rule_api_operations';
+import { useKibana } from '../../../../common/lib/kibana';
+import { CenterJustifiedSpinner } from '../../../components/center_justified_spinner';
+import { RuleEventLogListStatus } from './rule_event_log_list_status';
+
+const getParsedDate = (date: string) => {
+  if (date.includes('now')) {
+    return datemath.parse(date)?.format() || date;
+  }
+  return date;
+};
+
+const API_FAILED_MESSAGE = i18n.translate(
+  'xpack.triggersActionsUI.sections.ruleDetails.ruleEventLogListKpi.apiError',
+  {
+    defaultMessage: 'Failed to fetch event log KPI.',
+  }
+);
+
+const RESPONSE_TOOLTIP = i18n.translate(
+  'xpack.triggersActionsUI.sections.ruleDetails.ruleEventLogListKpi.responseTooltip',
+  {
+    defaultMessage: 'Response status of recent rule runs.',
+  }
+);
+
+const ALERTS_TOOLTIP = i18n.translate(
+  'xpack.triggersActionsUI.sections.ruleDetails.ruleEventLogListKpi.alertsTooltip',
+  {
+    defaultMessage: 'Alert status of recent rule runs.',
```

Comment:
Per above, I think "latest" is better than "most recent":
```suggestion
    defaultMessage: 'The alert statuses for the latest rule runs.',
```